### PR TITLE
Release 1.1.2

### PR DESCRIPTION
**Pull request by ricardopereira**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ We do not explicitly maintain compatibility with older platform versions; we no
 
 If you find any issues with unsupported platform versions, please [raise an issue](https://github.com/ably/ably-ios/issues) in this repository or [contact Ably customer support](https://support.ably.io) for advice.
 
-Note: As of version `1.1.1` this library based on the 1.1 library specification. It implements a subset of 1.1 features:
+Note: As of version `1.1.2` this library based on the 1.1 library specification. It implements a subset of 1.1 features:
 - updated push API and push device authentication;
 - support for enforcement of the maxMessageSize attribute
 Other minor features and bugfixes are included, as listed in the [changelog](CHANGELOG.md#110-2018-08-10).
```
[the remark this answers is not in the document]
Comment (ricardopereira) [reply README.md]: @paddybyers Well, the `set-version` script does that automatically.

[the remark this answers is not in the document]
Comment (paddybyers) [reply README.md]: @ricardopereira I meant the text "CHANGELOG.md#110-2018-08-10"

[the remark this answers is not in the document]
Comment (ricardopereira) [reply README.md]: Oh thanks! You're right. Will change.

[the remark this answers is not in the document]
Comment (ricardopereira) [reply README.md]: Done 7bebb84

**Comment by paddybyers** [comment]
OK, LGTM, thanks

**Pull request merged**